armoryengine/Transaction.py

```diff
@@ -430,6 +431,8 @@ def TxInExtractAddrStrIfAvail(txinObj):
       return hash160_to_addrStr( hash160(lastPush) )
    elif scrType == CPP_TXIN_SPENDP2SH:
       return binScript_to_p2shAddrStr(lastPush)
+   elif scrType == CPP_TXIN_P2WPKH_P2SH or scrType == CPP_TXIN_P2WSH_P2SH:
+      return binScript_to_p2shAddrStr(hash160(rawScript))
    else:
```

Comment:
I suppose so, but that might be really confusing for people as they won't necessarily expect to see an address there because technically there shouldn't.